.github/workflows/secrel4.yml

```diff
@@ -107,3 +110,21 @@ jobs:
       config-file: .github/secrel/config4.yml
       images: ${{ needs.images-for-secrel.outputs.all-images }}
     secrets: inherit
+
+  notify-secrel-error:
+    if: failure()
+    needs: [publish-images, images-for-secrel, secrel]
```

Comment:
`needs: secrel`
This step is notifying about SecRel failures and doesn't use output from other steps.